generic/tests/iozone_windows.py

```diff
@@ -19,27 +25,70 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment.
     """
-    vm = env.get_vm(params["main_vm"])
-    vm.verify_alive()
+
+    def post_result(results_path, analysisdir):
+        a = postprocess_iozone.IOzoneAnalyzer(list_files=[results_path],
+                                              output_dir=analysisdir)
+        a.analyze()
+        p = postprocess_iozone.IOzonePlotter(results_file=results_path,
+                                             output_dir=analysisdir)
+        p.plot_all()
+
+    def get_driver():
+        driver_name = params.get("driver_name", "")
+        drive_format = params.get("drive_format")
+        if not driver_name:
+            if "scsi" in drive_format:
+                driver_name = "vioscsi"
+            elif "virtio" in drive_format:
```

Comment:
@suqinhuang 
below update looks more clear and clean.

if "virtio" in driver_format:
     if "scsi" in driver_format:
           return "vioscsi"
     else:
           return  "viostor"
return None